README.md

```diff
@@ -32,7 +32,87 @@ def caller_two_from_a_separate_thread():
 def optional_init_function_to_prewarm():
     my_expensive_object()
 
+@once.once
+async def slow_expensive_object():
+    await load_expensive_async_resource()
+
+@once.once(per_thread=True)
```

Comment:
It would be good to clarify whether "per_thread" is really thread and not task (I think it is because it uses thread local storage, but people might wonder)

Reply:
done in https://github.com/DelfinaCare/once/pull/29